src/Replay/game/constants/Game.java

```diff
@@ -287,4 +287,91 @@
           put(10000, "VIRTUAL_OPCODE_LOGIN_REQUEST");
         }
       };
+
+  public static final class ItemAction {
+    public int itemId;
+    public int opcode;
+    public int containsWorldPoint;
+    public String name;
+
+    public ItemAction(int itemId, int opcode, int containsWorldPoint, String name) {
+      this.itemId = itemId;
+      this.opcode = opcode;
+      this.containsWorldPoint = containsWorldPoint;
+      this.name = name;
+    }
+  }
+
+  public static final Map<Integer, ItemAction> itemActionMap =
```

Comment:
Some of these names may change in the future. They don't really match up with what I have in the wireshark dissector.

Reply:
thats fine, whats there to do here?

Reply:
Possibly more English-looking names.